docs/strategies/circuit-breaker.md

```diff
@@ -37,6 +37,21 @@ new ResiliencePipelineBuilder().AddCircuitBreaker(new CircuitBreakerStrategyOpti
     ShouldHandle = new PredicateBuilder().Handle<SomeExceptionType>()
 });
 
+// Adds a circuit breaker with dynamic break duration:
+//
+// Same circuit breaking conditions as above, but with a dynamic break duration based on the failure count.
+// The duration is calculated as: minimum of (20 + 2^failureCount) seconds and capped at 400 seconds.
```

Comment:
I don't think these details are necessary, these are evident from the code. So maybe just keep the first line.